create_shards.c

```diff
@@ -534,3 +603,30 @@ IntegerToText(int32 value)
 
 	return valueText;
 }
+
+
+/*
+ *	GetSupportRoutine helps to find the support routine given a column, an access method
```

Comment:
`GetSupportRoutine` is a bad name because _routine_ isn't a word PostgreSQL uses frequently. I see _support function_ and _support procedure_, though the former seems more frequent, so let's go with that…

`GetSupportFunction` should be the name of this function.